pages/[lang]/contribute.tsx

```diff
@@ -92,12 +101,17 @@ const Community: NextPage = () => {
             />
           </Grid>
           <Grid item xs={12} md={6}>
-            <p>{t('contribute.sponsor.description')}</p>
-            <p>{t('contribute.sponsor.description2')}</p>
-            <p>
-              <Button href=''>{t('contribute.sponsor.membership')}</Button>{' '}
-              <Button href=''>{t('contribute.sponsor.donation')}</Button>
-            </p>
+            <p>{t('contribute.supporter.description')}</p>
+            <p>{t('contribute.supporter.sponsor')}</p>
+            <Button href='mailto:info@opentechschool.org'>
+              {t('contribute.supporter.contact')}
+            </Button>
```

Comment:
Should we put also another button or even a simple link to the about > membership section where they can read more about become a member? 🤔 
I'm not sure because there is not so much more explanation in the about page neither.

Reply:
yeah i think the link makes most sense